keepalived-vip/main.go

```diff
@@ -54,6 +50,8 @@ var (
 
 func main() {
 	clientConfig := kubectl_util.DefaultClientConfig(flags)
+
+	flags.AddGoFlagSet(flag.CommandLine)
```

Comment:
@bprashanth without this is not possible to use flags like `--v` to increase the verbosity of the logs